whylog/converters/__init__.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABCMeta, abstractmethod
+
+import dateutil.parser
+import six
+from frozendict import frozendict
+
+
+@six.add_metaclass(ABCMeta)
+class AbstractConverter(object):
+    @classmethod
+    @abstractmethod
+    def convert(cls, pattern_group):
+        pass
```

Comment:
Please raise exception here. Script execution should not reach this place. Function `convert` should be called only in subclasses.